dns/dyndns/src/etc/inc/plugins.inc.d/dyndns/phpDynDNS.inc

```diff
@@ -108,6 +112,7 @@
  *
  * DNSexit support and multiwan extension for pfSense by Ermal Luçi
  * Custom DNS support by Matt Corallo
+ * Cloudflare API token enhancements by Andreas Rupper
```

Comment:
I'm ok with credits but it has to be the copyright block at the top

Reply:
I don't insist on that. can be removed.